app/src/main/res/values/strings.xml

```diff
@@ -2,7 +2,7 @@
 <resources>
   <string name="app_name">Oppia</string>
   <string name="nav_header_title">Sean</string>
-  <string name="nav_header_subtitle">14 Stories Completed | 3 Topics In Progress</string>
+  <string name="nav_header_subtitle">%s Stories Completed | %s Topics In Progress</string>
```

Comment:
Can we have plural form of this story count.

Reply:
Converted this into a multiple plurals string